app/models/review_bid.rb

```diff
@@ -21,29 +21,29 @@ def self.bidding_data(assignment_id, reviewer_ids)
   end
 
   # assigns topics to reviews as matched by the webservice algorithm
-  def self.assign_review_topics(assignment_id, reviewer_ids, matched_topics, _min_num_reviews = 2)
+  def assign_review_topics(assignment_id, reviewer_ids, matched_topics, _min_num_reviews = 2)
```

Comment:
I'm confused about why there's a method to do that in this class, as well as in the review_bids_controller.  In general, changes to the model should be in a model class, so maybe functionality should be moved from the controller to the model.

Reply:
Also, it doesn't look like an instance method.  It doesn't relate to a single review bid.